sys_arm64.s

```diff
@@ -6,7 +6,7 @@
 #include "textflag.h"
 #include "go_asm.h"
 #include "funcdata.h"
-#include "internal/abi/abi_arm64.h"
+#include "abi_arm64.h"
```

Comment:
Is this even used in this file? Can it be removed?

Reply:
You are right this is not needed here
```suggestion
```